File: Easy/Java/961. N-Repeated Element in Size 2N Array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
        Map<Integer, Integer> appearance = new HashMap();
        int num = 0;

        for(int i : nums){
            if(appearance.containsKey(i))
                appearance.put(i, appearance.get(i) + 1);
            else
                appearance.put(i, 1);
        }

        for(int i : nums){
            if(appearance.get(i) == (nums.length / 2)){
                num = i;
                break;
            }
        }

        return num;
    }
}

File: Easy/Python/961. N-Repeated Element in Size 2N Array.py
class Solution(object):
    def repeatedNTimes(self, nums):
        appearance = {}
        num = 0

        for i in nums:
            if i in appearance:
                appearance[i] = appearance[i] + 1
            else:
                appearance[i] = 1
         
        for i in nums:
            if appearance[i] == len(nums) / 2:
                num = i
                break
        
        return num
